templates/web/base/report/new/category_extras.html

```diff
@@ -15,12 +15,16 @@
     <div class="extra-category-questions">
         [% UNLESS category_extras_notices.$category %]
           <h2 class="visuallyhidden form-section-heading">[% loc('Extra details') %]</h2>
+          [% TRY %]
+            [% INCLUDE 'report/new/_category_extras_header.html' %]
```

Comment:
The message won't be shown unless a category is picked that has extra questions, and not all of Bexley's do - e.g. "Damage to grass verges", "Vehicles for sale on the public road", Animal fouling -> Park / open space, or Flytipping -> Garage areas.
I guess instead of as code it could be added as a 'notice' on every category, would that work the same? Bit annoying if they want to e.g. update the phone number I guess, but consistently displayed then. Otherwise, not sure - `report/new/form_after_heading.html`?

Reply:
I admit I was implementing their wireframes fairly literally, and assumed it would be OK to only appear on categories that already had extra fields...

Might be a bit tedious to put it on every category and manage manually. Maybe added in code in `Bexley::munge_report_new_category_list`?

Reply:
Though it occurred to me that doing it in the template will restrict the warning to the Bexley cobrand (desirable, I think?), whereas doing it as notices will put it on FMS.com and the app (which I don't think we want?)

Reply:
Yeah, good point. Wonder where the best place to put it in a template is, then, if we want it for all categories.

Reply:
I've asked the client if this would be OK, would make it a much simpler change:

<img width="663" alt="image" src="https://user-images.githubusercontent.com/4776/157218115-5fd31c1f-a32d-4eb3-8cc2-53ac042e10eb.png">